superset-frontend/src/dashboard/stylesheets/dashboard.less

```diff
@@ -60,6 +60,7 @@ body {
   position: relative;
   font-size: @font-size-l;
   font-weight: @font-weight-bold;
+  margin-bottom: 3px;
```

Comment:
![image](https://user-images.githubusercontent.com/335541/75829755-e4cce480-5d63-11ea-9cd4-f559b616a41f.png)

This adds a little space between dashboard chart container header and visualizations.